src/Activator.php

```diff
@@ -27,5 +29,33 @@ protected function hooks() {
 	 */
 	public static function run(): void {
 		Campaigner::register_role_and_add_capabilities();
+		self::do_migrations();
+	}
+
+	private static function do_migrations(): void {
+		ini_set( 'display_errors', 1 );
+		// Fetch migration ids that have run from WP option.
+
+		$log = MigrationLog::from_wp_options();
+
+		/**
+		 * @var Migration[] $migrations
+		 */
+		$migrations = [
+			M001EnableEnFormFeature::class,
```

Comment:
We might want to replace this with something that automatically loads all files in the `\Migrations` namespace.

Reply:
Just mentioning Flo's branch here too which does exactly that :100: https://github.com/greenpeace/planet4-master-theme/commit/aefab111da63d2a25e0525f1576580baeb95746a